addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -273,7 +275,7 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
      */
     _setEmoji: function (id, unicode) {
         const emojis = this.$(`.o_article_emoji_dropdown[data-article-id="${id}"] > .o_article_emoji`);
-        emojis.text(unicode || '📄');
+        emojis.text(unicode || '');
```

Comment:
I would the the opportunity to rename "'unicode"' that does not mean much. Unicode is an encoding, not a source string of anything related ^^ .